tests/ci_build/Dockerfile.cpu.clang

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+COPY install/ubuntu_install_core.sh /install/
+RUN /install/ubuntu_install_core.sh
+COPY install/ubuntu_install_python.sh /install/
+RUN /install/ubuntu_install_python.sh
+COPY install/ubuntu_install_scala.sh /install/
+RUN /install/ubuntu_install_scala.sh
+COPY install/ubuntu_install_r.sh /install/
+RUN /install/ubuntu_install_r.sh
+COPY install/ubuntu_install_perl.sh /install/
+RUN /install/ubuntu_install_perl.sh
+
+# Install clang 3.9 (the same version as in XCode 8.*)
+RUN apt-get -y install clang
```

Comment:
Might think about version binding? I'm not familiar with the clang compiler, but if it's usually no problem to simply upgrade I'd be happy to leave it as-is

Reply:
You mean pinning it to a specific version right?  I actually had a specific version specified here, I can re-enable it.  Believe it or not the public key from llvm.org was returning a bunch of server errors the other day.  It seems fine today, so let's try and re-enable it and keep an eye out for server errors.